config/routes.rb

```diff
@@ -47,6 +47,7 @@
   resources :sections, only: [:index, :show]
   resources :import_records, only: [:index]
   resources :iep_documents, only: [:show]
+  resources :school_administrator_dashboard, only: [:show]
```

Comment:
A Rails-y pattern here would be to have `administrator_dashboard` as a `member` route on the school resource.